web/modules/custom/ai_screening_project/ai_screening_project.module

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Hooks related to this module.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+
+/**
+ * Implements hook_entity_base_field_info().
+ */
+function ai_screening_project_entity_base_field_info(EntityTypeInterface $entityType): array {
+  $fields = [];
+
+  if ($entityType->id() === 'node') {
+    $fields['corrupted'] = BaseFieldDefinition::create('boolean')
```

Comment:
Consider defining a `FIELD_CORRUPTED`, say, constant for the field name (`'corrupted'`) in a `NodeHelper` class. This class can also have useful function for checking the field, e.g.

```
class NodeHelper {
  public const FIELD_CORRUPTED = 'corrupted';

  public function isCorrupted(NodeInterface $node): bool;

  public function setCorrupted(NodeInterface $node, bool $corrupted = TRUE): void;

  …
} 
```

Reply:
This can be rewritten when switching to events